code/game/objects/items/storage/firstaid.dm

```diff
@@ -280,14 +368,14 @@
 /obj/item/storage/firstaid/tactical/PopulateContents()
 	if(empty)
 		return
-	new /obj/item/stack/medical/gauze(src)
-	new /obj/item/defibrillator/compact/combat/loaded(src)
-	new /obj/item/reagent_containers/hypospray/combat(src)
-	new /obj/item/reagent_containers/pill/patch/styptic(src)
-	new /obj/item/reagent_containers/pill/patch/styptic(src)
-	new /obj/item/reagent_containers/pill/patch/silver_sulf(src)
-	new /obj/item/reagent_containers/pill/patch/silver_sulf(src)
-	new /obj/item/clothing/glasses/hud/health/night(src)
+	var/static/items_inside = list(
+		/obj/item/stack/medical/gauze = 1,
+		/obj/item/defibrillator/compact/combat/loaded = 1,
+		/obj/item/reagent_containers/hypospray/combat = 1,
+		/obj/item/reagent_containers/pill/patch/styptic = 2,
+		/obj/item/reagent_containers/pill/patch/silver_sulf = 2,
+		/obj/item/clothing/glasses/hud/health/night = 1)
```

Comment:
Does this overfill the medkit, since a gauze is being added

Reply:
oops, wrong math, yeah i can remove the gauze since it's quite a common find anyway